ydb/library/login/login.h

```diff
@@ -216,6 +218,8 @@ class TLoginProvider {
     static std::chrono::system_clock::time_point GetTokenExpiresAt(const TString& token);
     static TString SanitizeJwtToken(const TString& token);
 
+    TString GenerateHash(const TString& password);
```

Comment:
I do not like that you create this interface function only for tests. I suggest to create hash and hardcode it to test and do not use function  `GenerateHash` from login provider